Countries/FirstScreen/ListViewController.swift

```diff
@@ -15,14 +15,29 @@ class ListViewController: UITableViewController {
     
     var countries: [CountriesApiQuery.Data.Country] = []
     
+    let refrechControll: UIRefreshControl = {
+        let refrechControll = UIRefreshControl()
+        return refrechControll
+    }()
+
+    @objc private func refresh(sender: UIRefreshControl) {
+        loadData(sender: sender)
+    }
```

Comment:
Do you need this function, as it only pass argument to the next one?

Reply:
yes.. i mean i dont need it at all